tests/console/zypper_in.pm

```diff
@@ -75,4 +75,12 @@ sub run {
     }
 }
 
+
+sub post_fail_hook {
+    my $self = shift;
+
+    upload_logs "/var/log/zypper.log";
```

Comment:
maybe also add ` /var/log/history`

Reply:
not sure what this should contain - but it doesn't exist on my local tumbleweed system